apps/demos/Demos/Splitter/Overview/React/App.tsx

```diff
@@ -15,7 +24,7 @@ const App = () => (
         resizable={true}
         size="140px"
         minSize="70px"
-        render={PaneContentWithTitleRender('Left Pane', '140px')}
+        template={(data,index,element) => generate(data,element,'Left Pane', '140px')}
```

Comment:
we cannot use `template` property here, it's just not a React-way. In React we are supposed to use only `component` or `render` options, but both of it does not contain `element` as a parameter.

Seems like our goal can be achieved only using `onItemRendered`

Reply:
[80ddd95](https://github.com/DevExpress/DevExtreme/pull/27306/commits/80ddd957e53091fd0528eccd3f69be639215fa14)